src/debugger/adapter/DebugAdapter.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
```

Comment:
nit: I see this project is setup with `prettier` in the `devDependencies` - but don't see the `@format` --> did you use prettier to format this file?

Reply:
Our prettier seems to run off of directory structure instead of @format annotations, everything matching `src/**/*.js` or `scripts/**/*.js` should get formatted.